### DISCUSSION TITLE
mermaid syntax error

### DISCUSSION THREAD

**Discussion by jdavidrcamacho** [zauberzeug/nicegui#2390] [Ideas / Feature Requests]
### Question

Hello,

Is there a way for if a mermaid diagram is wrong, e.g,

```py
from nicegui import ui  

ui.mermaid('''
graph LR;
    A --> B;
    A -> C;
''')
ui.run()
```

Instead of showing in my browser the "syntax error" page, it shows a error message in my terminal?

**Comment by falkoschindler** [comment]
Hi @jdavidrcamacho!

Currently that's not possible. We would need to catch the error in JavaScript and send an event to the server. But it shouldn't be too hard.

**Reply by falkoschindler** [reply]
I just created PR #2396 with this feature.

**Reply by jdavidrcamacho** [reply]
Thanks!!!
